tests/test_wiki_page.py

```diff
@@ -19,6 +19,33 @@ def test_edit_wiki_page(self):
             content, self.subreddit.get_wiki_page('index', u=1).content_md)
 
     @betamax
+    def test_edit_wiki_page_settings(self):
+        self.r.login(self.un, self.un_pswd, disable_warning=True)
```

Comment:
This shouldn't be necessary as `betamax_init` should already do it. It doesn't really hurt to have it though.